src/ui-kit/InnerTable/InnerTableBody.tsx

```diff
@@ -0,0 +1,52 @@
+import { JSX } from '@redneckz/uni-jsx';
+import { type UniBlockProps } from '../../types';
+import { Img } from '../Img/Img';
+import { InnerTableBodyItem } from './InnerTableBodyItem';
+import { DataItem } from './InnerTableProps';
+
+export interface InnerTableBodyProps extends UniBlockProps {
+  activeTabIndex: number;
+  items: DataItem[];
+}
+
+export const InnerTableBody = JSX<InnerTableBodyProps>(({ context, activeTabIndex, items }) => {
+  const [hiddenTables, setHiddenTables] = context.useState({});
```

Comment:
State не типизирован. Надо через дженерик задать типа, обязательно.
```suggestion
  const [hiddenTables, setHiddenTables] = context.useState<ТипЭтихДанных>({});
```